spread.yaml

```diff
@@ -17,7 +17,8 @@ backends:
       echo "Allocating $SPREAD_SYSTEM..."
       docker_image=$(echo $SPREAD_SYSTEM | awk -F '-' '{print $1":"$2}')
       docker_arch=$(echo $SPREAD_SYSTEM | awk -F '-' '{print $NF}')
-      docker run --rm -e DEBIAN_FRONTEND=noninteractice \
+      docker run --cap-add SYS_ADMIN --security-opt apparmor=unconfined \
```

Comment:
as for the others and previous discussions, we'll need to disallow this. can you test these slices with something more basic?

Reply:
I dropped the added capabilities and now just use LXD backend, it passes there